### DISCUSSION TITLE
Allow OAuth2 authentication for external Alertmanager

### DISCUSSION THREAD

**Discussion by jstaffans** [grafana/grafana#48892] [Alerting]
It seems that if you configure an external Alertmanager when using Unified Alerting, the only authentication method available is Basic Auth. Is it planned to add OAuth2 authentication as well? 

Prometheus does for example support using OAuth2 when configuring its Alertmanager integration: https://prometheus.io/docs/prometheus/latest/configuration/configuration/#oauth2

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
